src/commands/node/install.ts

```diff
@@ -1,17 +1,38 @@
 import { SwankyCommand } from "../../lib/swankyCommand.js";
-import { ux } from "@oclif/core";
+import { ux, Flags } from "@oclif/core";
 import { downloadNode, swankyNode } from "../../lib/index.js";
 import path from "node:path";
 import { writeJSON } from "fs-extra/esm";
+import inquirer from "inquirer";
+import { DEFAULT_NODE_INFO } from "../../lib/consts.js";
+import { pickNodeVersion } from "../../lib/prompts.js";
 
 export class InstallNode extends SwankyCommand<typeof InstallNode> {
   static description = "Install swanky node binary";
 
+  static flags = {
+    "set-version": Flags.string({
+      description: "Specify version of swanky node to install",
+      required: false,
+    }),
+  }
   async run(): Promise<void> {
     const { flags } = await this.parse(InstallNode);
     if (flags.verbose) {
       this.spinner.verbose = true;
     }
+    let nodeVersion= DEFAULT_NODE_INFO.version;
+
+    if (flags.specifyVersion) {
```

Comment:
Flag name is "set-version"

Reply:
Also make sure to add a check to ensure the provided version is a supported one. For example:

`../bin/run.js node install --set-version=1.7.0`

Should throw an appropriate **InputError** message containing also the list of available versions. The supported versions must be list in the command description too.